### PR TITLE
Update wdio-wikibase in Docker/test/selenium to 5.1.0

### PR DESCRIPTION
**Pull request by dati18**
Bug: T288443

https://phabricator.wikimedia.org/T288443

### CODE REVIEW

File: Docker/test/selenium/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
 		"npm-run-all": "^4.1.5",
 		"stylelint-config-wikimedia": "0.10.1",
 		"wdio-mediawiki": "^1.0.0",
-		"wdio-wikibase": "^4.0.1",
+		"wdio-wikibase": "^5.1.0",
 		"webdriverio": "^6.1.16",
```
Comment (toban) [inline Docker/test/selenium/package.json:23]: I think this line needs to go, it seems to not be picking the correct webdriverio version

Comment (toban) [reply Docker/test/selenium/package.json:23]: https://github.com/wmde/wikibase-release-pipeline/pull/211/checks?check_run_id=3484803294#step:4:284

**Review by toban — CHANGES_REQUESTED** [review]
Found some weird stuff going through the logs, looks like it's picking the wrong version after all.

File: Docker/test/selenium/package.json [hunk 2]
```diff
@@ -20,7 +20,6 @@
 		"stylelint-config-wikimedia": "0.10.1",
 		"wdio-mediawiki": "^1.0.0",
 		"wdio-wikibase": "^5.1.0",
-		"webdriverio": "^6.1.16",
```
Comment (toban) [inline Docker/test/selenium/package.json:23]: https://github.com/wmde/wikibase-release-pipeline/pull/211/checks?check_run_id=3487116732#step:4:1307

still there

Comment (dati18) [reply Docker/test/selenium/package.json:23]: I found this in `package-lock.json`. I think it's the reason why `webdriverio-6.8.0` appeared in the test

Screenshot from 2021-09-02 12-36-26

Comment (dati18) [reply Docker/test/selenium/package.json:23]: line 6902

Comment (toban) [reply Docker/test/selenium/package.json:23]: I think what was required is this -> https://github.com/wmde/wikibase-release-pipeline/pull/212

**Pull request merged**